sklearn/tests/test_naive_bayes.py

```diff
@@ -214,15 +214,17 @@ def test_discretenb_deprecated_coef_intercept(cls):
             hasattr(est, att)
 
 
-@pytest.mark.parametrize("cls", [MultinomialNB, BernoulliNB, CategoricalNB])
+@pytest.mark.parametrize("cls", [MultinomialNB, ComplementNB, BernoulliNB,
+                                 CategoricalNB])
```

Comment:
@glemaitre, there are 2 tests that run on all five naive Bayes classes and 8 more than run on the four discrete naive Bayes classes. I thought it might be nice to simplify the code by defining lists up at the top:
```
DISCRETE_NAIVE_BAYES_CLASSES = [BernoulliNB, CategoricalNB, ComplementNB, MultinomialNB]
ALL_NAIVE_BAYES_CLASSES = DISCRETE_NAIVE_BAYES_CLASSES + [GaussianNB]
```
and then using these in the `@pytest.mark.parametrize('class', ...)` calls. I'd be happy to implement, but I wanted to check whether constructing an explicit list for each test is preferred?

Reply:
To be explicit, this is what I had in mind: https://github.com/dpoznik/scikit-learn/commit/2c548154420feab14fb322d6076a7bef7c8b7c35

Reply:
This would be fine with me. We tend to do that in some tests.

Reply:
Cool, thanks for confirming.